{
  "issue_type": "bug",
  "extracted_from": "/root/yunwei37/vllm-exp/bug-study/analysis_llm/sample_results/vllm/label_based/bug/issues.json",
  "issue": {
    "number": 4081,
    "title": "[Bug]: Error when running pytest: TypeError: 'ABCMeta' object is not subscriptable",
    "body": "### Your current environment\n\nWhen calling `python collect_env.py`, running into the same error as below.\r\n\n\n### \ud83d\udc1b Describe the bug\n\nEncountering the below issue when running tests. Suspected to be related to mypy changes\r\n![image](https://github.com/vllm-project/vllm/assets/88394319/fd9057d8-6f9c-4f51-b60d-88b7b88b70d2)\r\n",
    "labels": [
      "bug"
    ],
    "state": "closed",
    "created_at": "2024-04-15T06:43:33+00:00",
    "closed_at": "2024-04-15T21:47:32+00:00",
    "comments": 4,
    "reactions": {
      "url": "https://api.github.com/repos/vllm-project/vllm/issues/4081/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "author_association": "CONTRIBUTOR",
    "html_url": "https://github.com/vllm-project/vllm/issues/4081"
  }
}